### DISCUSSION TITLE
Proposal: Update Wiki images with the current UI - including the dark theme option

### DISCUSSION THREAD

**Discussion by u-RraaLL** [uBlockOrigin/uBlock-issues#2033] [Ideas]
There's quite a bit of outdated images across the wiki. I'm proposing updating them and serving a 2nd (dark theme) option as well.

There's no hurry, so I can take it upon myself, if needed.

As per [docs](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to) it should be possible to use github theme-based variants. Unfortunately, when I tried using it on the draft wiki, it didn't seem to work when previewing. (Working fine when testing on a new page on a fork.)
I'm not sure if this requires toggling some setting in the project.

For the time being the theme-based images aren't yet supported in the README, but the issue about this has already moved to the backlog so it will be fixed at some point.

For now, I'm proposing: https://github.com/gorhill/uBlock/compare/master...u-RraaLL:patch-1

**Comment by u-RraaLL** [comment]
Any opinion @gorhill?

Is there a way to enable theme-based images in the wiki or should I just add them side by side, like in my README suggestion patch?

**Comment by gorhill** [comment]
We should stick to light theme by default for consistency. We can update images for when there are new features present in current versions of uBO. We want to document functionality, not appearance -- which is just pixels colored a different way.

**Reply by u-RraaLL** [reply]
That's a shame. I was hoping to stop getting blinded when scrolling the wiki at night. Oh, well~

There's still plenty of images with outdated UI across the wiki though. Perhaps the Report button doesn't need to be present in every single screenshot\*, but there are still some pages using screenshots as old as 2015 (looking at the article dates).
BTW - is the 3p script/frame mini-filtering widget (in the popup) even present in the wiki? It's been almost a year since the feature's addition...

\* Not wiki, ok, but I still think the button should make it onto the README images.

**Comment by gorhill** [comment]
> I was hoping to stop getting blinded when scrolling the wiki at night.

The light theme version is still shown in your changes. I don't mind pulling in your changes for the popup panel, my point is that we shouldn't extend everywhere showing two versions of the user interface every time we document features.

**Reply by u-RraaLL** [reply]
Well, like I originally mentioned I wanted to make use of the theme based images, which aren't yet available for the README.

And I certainly wasn't going to remove the light theme altogether without approval. But if you're open to the idea, I could certainly try replacing all images with the dark theme equivalent. Though that's not likely what you meant.

I understand the point about not having twice the work, but I don't think new features are introduced often enough to cause problems, plus, [github's theme-based images](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to), which is still what I'm referring to, won't be applied if the suffix isn't present. -> New features could be added for a single theme, without the suffix, until some volunteer wants to add the 2nd option.

Edit:

> The light theme version is still shown in your changes.

Also, at worst I could always network block them - isn't that what uBO is for?
